MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -28,24 +28,24 @@
             <div class="card-body" style="background-color:white">
                 <h2 class="card-title">Mohini Chakravarty</h2>
                 <p class="card-text">
-                    Scrum Master
+                    <div> Scrum Master </div> 
                 </p>
             </div>
         </div>
         <div class="col-sm-5">
-            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Mohini">
+            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Image of Mohini, the scrum master of the team" />
```

Comment:
made the alt tag more descriptive, so that if the image does not load, the user is able to see the description and understand what the image is all about.